src/styles/normalize.scss

```diff
@@ -0,0 +1,12 @@
+* {
```

Comment:
Rename this file as what is done here is not `normalising` but rather `resetting`,